### PR TITLE
[WEBSITE] Add blog post for deprecating Python installer

### PR DESCRIPTION
**Pull request by hshin-g**
Adding blog post for deprecating Python installer.

Links to #3166.

### CODE REVIEW

File: _news/2019-09-17-deprecate-python-installer.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Terraform: Official Installation Path of Forseti Security
+author: Hannah Shin
+---
+
+Forseti Community,
+
+We’ve heard from many of our users that Forseti’s Deployment Manager-based
```
Comment (blueandgold) [inline _news/2019-09-17-deprecate-python-installer.md]: Let's avoid the negativity for DM, how about something like this?
```
We’ve heard from many of our users that we should focus on delivering
a faster, simpler, and more consistent deployment experience with Terraform. 
```

File: _news/2019-09-17-deprecate-python-installer.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+be deprecated effective October 3, 2019, and will no longer be available as an 
+installation option for future releases. 
+
+We have created a migration script and documentation to help you seamlessly 
+migrate from Deployment Manager to Terraform:
+
+* For v2.18.0+ users, refer to the instructions to migrate 
+[here]({% link _docs/latest/setup/migrate.md %}).
+
+* For versions earlier than v2.18.0, please upgrade to v2.18.0 through
```
Comment (blueandgold) [inline _news/2019-09-17-deprecate-python-installer.md]: What would be the amount of work to have to migration script cover to v2.13 (last 10 versions)?

File: _news/2019-09-17-deprecate-python-installer.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+Forseti Community,
+
+We’ve heard from many of our users that Forseti’s Deployment Manager-based 
+upgrade path was slow and inefficient, so we're focusing on delivering a 
+faster, simpler, and more consistent experience with Terraform. 
+
+Starting with Forseti Security v2.22.0, Forseti will support Terraform as the 
+official installation path. The Python based Deployment Manager installer will
```
Comment (blueandgold) [inline _news/2019-09-17-deprecate-python-installer.md]: nit: add a hyphen `Python-based`

File: _news/2019-09-17-deprecate-python-installer.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Forseti Community,
+
+We’ve heard from many of our users that Forseti’s Deployment Manager-based 
+upgrade path was slow and inefficient, so we're focusing on delivering a 
+faster, simpler, and more consistent experience with Terraform. 
+
+Starting with Forseti Security v2.22.0, Forseti will support Terraform as the 
+official installation path. The Python based Deployment Manager installer will 
+be deprecated effective October 3, 2019, and will no longer be available as an
```
Comment (blueandgold) [inline _news/2019-09-17-deprecate-python-installer.md]: nit: how about moving the date `October 3, 2019` next to `v2.22.0` earlier in this paragraph?  seems to flow better

File: _news/2019-09-17-deprecate-python-installer.md (hunk 1 above)
Comment (hshin-g) [reply _news/2019-09-17-deprecate-python-installer.md]: How about:

```We’ve heard feedback from many of our users around the current Forseti installation and upgrade path, and we are focusing on delivering a faster, simpler, and more consistent experience with Terraform. ```

File: _news/2019-09-17-deprecate-python-installer.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+[here]({% link _docs/latest/setup/migrate.md %}).
+
+* For versions earlier than v2.18.0, please upgrade incrementally to v2.18.0 through 
+Deployment Manager following the steps [here]({% link _docs/latest/setup/upgrade.md %}) 
+first before migrating.
+
+You can learn more about the Forseti Terraform module source code 
+[here](https://registry.terraform.io/modules/terraform-google-modules/forseti/google/).
+
+Please reach out to us on Slack or discuss@forsetisecurity.org with any questions.
```
Comment (blueandgold) [inline _news/2019-09-17-deprecate-python-installer.md]: nit: Can we make `Slack` a link here to our actual slack #general channel?

**Pull request merged**